quantum/keycode_config.c

```diff
@@ -88,3 +88,32 @@ uint16_t keycode_config(uint16_t keycode) {
             return keycode;
     }
 }
+
+uint8_t mod_config(uint8_t mod) {
+    if (keymap_config.swap_lalt_lgui) {
+        if (mod & MOD_LGUI) {
+            mod &= ~MOD_LGUI;
+            mod |= MOD_LALT;
+        }
+        if (mod & MOD_LALT) {
```

Comment:
I just took a very quick look, and I happened to notice this.

Changing this to `else if` should fix the issue. The block below also has the same issue so it should be changed too.